### DISCUSSION TITLE
stack original and edited versions

### DISCUSSION THREAD

**Discussion by Qhilm** [simulot/immich-go#489] [Q&A]
I noticed that edited pictures are stored as a separate picture in google photos, and end up being detected as duplicate in Immich, after uploading them with immich-go. Is there a way to automatically stack edited and original pictures upon uploading to Immich?

**Comment by simulot** [comment]
Not at the moment.
But this is a nice suggestion

**Comment by Qhilm** [comment]
Already tracked in https://github.com/simulot/immich-go/issues/162
